crates/bevy_sprite/src/lib.rs

```diff
@@ -137,7 +136,7 @@ pub fn calculate_bounds_2d(
     sprites_to_recalculate_aabb: Query<
         (Entity, &Sprite, &Handle<Image>, Option<&TextureAtlas>),
         (
-            Or<(Without<Aabb>, Changed<Sprite>)>,
+            Or<(Without<Aabb>, Changed<Sprite>, Changed<TextureAtlas>)>,
```

Comment:
wouldn't that make the `TextureAtlas` component mandatory?

Reply:
I don't think so, it's in a `Or` filter